src/components/BoxApproval/BoxApproval.jsx

```diff
@@ -141,16 +141,22 @@ const BoxApproval = () => {
         </Tabs>
         <div className={styles.legend}>
           <div className={styles['request-changes-row']}>
-            <BsFillArrowRightCircleFill className={styles['request-changes-icon']} />
-            <p className={styles['request-changes-text']}>Request Changes</p>
+            <div className={styles['request-changes-row-content']}>
+              <BsFillArrowRightCircleFill className={styles['request-changes-icon']} />
```

Comment:
This should be below Found a Box and Launched icons. If you change the legend's styling to flex box as mentioned in the previous comment, then the ordering will flip back to how it is in the jsx. Let me know if y'all have questions about the css styling here.

Reply:
Just noticed that the ordering of the legend icons on mobile is different from desktop. Please use the isMobile hook to render the request changes icon in the right place for each view. You can also ask design if it's okay to place these icons in the same order for desktop and mobile view instead.